### ISSUE TITLE
Some icons are not visible at status bar

### ISSUE THREAD

**Issue by vlad-gad** [aziz/PlainTasks#265]
Hi guys,
Many thanks for this awesome plugin. I use it every day in my dev projects. It's extremely helpful :)

I noticed this little issue when I wanted to change the message in the status bar to:

`{ "stats_format": "Done: $n/$a → ☐: $o | ✔: $d | ✘: $c | Progress: $percent% $progress | Last @done: $last :: " }`

But unfortunately I can't see the 3 icons.

I tried to put this:

`{ "stats_format": "❍ ❑ ☐ ✔ ✘ ☑ ✓ # -  ■ □ ▪ ▫ – — ≡ → › [ ] | àéùîï@€ абвгдежйъ |" }`

and I can see only the characters after the `#` sign, not before him.

Do you have any idea why I have problem with this 7 icons?

Many thanks

**Comment by vovkkk** [comment]
It depends on OS and font of UI.
You may have problem because of some bug (either ST or OS) or because your system lacks full Unicode support.

All chars are visible on Windows 8.1 Enterprise

![2016-02-27 22-30-40](https://cloud.githubusercontent.com/assets/1975298/13374025/91239184-dda2-11e5-974f-c71d51fef7f0.png)

As workaround try to use square root `√` and multiplication sign `×`.

**Comment by vlad-gad** [comment]
Yeah, on Win 7 & 8.1 it works well but not on Win 10. I tried to copy all fonts from a Win 8.1 PC and also set the same language settings like this machine but without result.

Can you tell me what is your option to:

`Control Panel` > `Clock, Language and Region` > `Region` > `Administrative tab` -- `Current language for non-Unicode programs`

Is there any program to install that can fix this? Or any setting/hack to set?

**Comment by vovkkk** [comment]
> not on Win 10

All right in virtual machine

![2016-02-29 14-44-13 w10ef - oracle vm virtualbox](https://cloud.githubusercontent.com/assets/1975298/13390018/c8ec933c-def5-11e5-80d5-37c9fe7976f3.png)

Do you perhaps have increased scale in monitor settings? it is known ST bug https://github.com/SublimeTextIssues/Core/issues/1066



> Current language for non-Unicode programs

Russia(n)
